addons/mail/static/src/core/web/mail_composer_attachment_list.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates>
+    <t t-name="mail.MailComposerAttachmentList">
+        <ul class="list-unstyled">
+            <li t-foreach="files" t-as="file" t-key="file_index" class="d-flex align-items-center bg-100 mw-50 p-1 ps-3 my-2">
+                <a t-att-href="getUrl(file.id)" t-out="file.name" class="flex-grow-1 text-truncate"/>
+                <button class="btn flex-shrink-0" t-on-click.stop="() => this.onFileRemove(file.id)">
+                    <i class="fa fa-w fa-times"/>
```

Comment:
`fa-w` => `fa-fw` I suppose?

Also `fa fa-times` can be replaced by prettier `oi oi-close`.

Reply:
I noticed that the widget to add new recipients uses `fa-times` for the icon allowing people to delete a recipient from the list (see: the tag). For consistency, I will keep using `fa-times`.

Reply:
Most `fa-times` that have been kept could be adapted to `oi-close`. The odoo way is mostly adapt when we feel like it, even if there are some inconsistencies at some point.

All in all it doesn't matter much, probably we'll make a pass for that in the future ;)